astropy/units/physical.py

```diff
@@ -7,8 +7,15 @@
 
 from astropy.utils.exceptions import AstropyDeprecationWarning
 
-from . import imperial  # noqa: F401  # Needed for backward namespace compat, see #11975 and #11977
-from . import astrophys, cgs, core, misc, quantity, si
+from . import (  # noqa: F401  # Needed for backward namespace compat, see #11975 and #11977
+    astrophys,
+    cgs,
+    core,
+    imperial,
```

Comment:
Can you move the comment back to `imperial`?

Reply:
Done